src/System.Management.Automation/engine/remoting/client/Job.cs

```diff
@@ -3824,7 +3824,7 @@ private void HandleRunspaceAvailabilityChanged(object sender, RunspaceAvailabili
         #region Private Members
 
         // helper associated with this job object
-        private RemotePipeline _remotePipeline = null;
+        private readonly RemotePipeline _remotePipeline = null;
```

Comment:
We can remove the null assignment as it is unneeded.

Reply:
We could do this in separate PR, clean all code base and force a rule for this.